coordinator/stargate-starter/src/main/java/io/stargate/starter/Starter.java

```diff
@@ -317,8 +326,8 @@ private static void logerr(String message, Level level) {
   }
 
   protected void setStargateProperties() {
-    if (version == null || version.trim().isEmpty() || !NumberUtils.isParsable(version)) {
-      throw new IllegalArgumentException("--cluster-version must be a number");
+    if (version != null) {
+      log("ignoring deprecated option --cluster-version", Level.WARNING);
```

Comment:
Should we also specify to use `persistence-module` instead?

Reply:
ok